tests/assets/puppet-control-repo/hiera.yaml

```diff
@@ -0,0 +1,12 @@
+---
+version: 5
```

Comment:
this file is not used by our tests

it, and all files in `data` can be omitted